packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -1,26 +1,20 @@
 ---
-title: Submit an SSO integration with the OIN Wizard
+title: Submit an integration with the OIN Wizard
 meta:
   - name: description
     content: Learn how to submit your integration to the Okta Integration Network (OIN) team for publication. The submission task is performed in the Okta Admin Console through the OIN Wizard.
 layout: Guides
 ---
 
-Use this guide to learn how to submit an SSO integration to the Okta Integration Network (OIN) team using the OIN Wizard.
+Learn how to submit an OIDC, SAML 2.0, or SCIM 2.0 integration to the Okta Integration Network (OIN) using the OIN Wizard.
 
 ---
 
-#### Learning outcomes
-
-* Learn how to submit an SSO integration using the OIN Wizard.
-* Learn how to test your SSO integration with the OIN Submission Tester before submission.
-* Learn how to update a published SSO integration.
-
 #### What you need
 
 * An [Okta Developer Edition org](https://developer.okta.com/signup/). The OIN Wizard is only available in Okta Developer Edition orgs.
 * An admin user in the Okta Developer Edition org with either the super admin or the app and org admin roles
-* A functional SSO integration based on the [Build a Single Sign-On integration](/docs/guides/build-sso-integration/) guide
+* A functional integration based on the [Build a Single Sign-On integration](/docs/guides/build-sso-integration/) or [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview/main/) guide
```

Comment:
you can have an integration with both SCIM and SSO

Reply:
Fair enough - but this has an "or" in there, suggesting that it's either one or the other. 

You could do maybe:
A functional integration. This can be based on the [Build SSO] guide, the [Build SCIM] guide, or both types.